lib/nidobata.rb

```diff
@@ -15,11 +15,20 @@ def init
       email = ask('Email:')
       password = ask('Password:', echo: false)
       data = {grant_type: 'password', username: email, password: password}
-
-      token = JSON.parse(http.post('/oauth/token', data.to_json, {'Content-Type' => 'application/json'}).tap(&:value).body)['access_token']
-      netrc = Netrc.read
-      netrc[IDOBATA_URL.host] = email, token
-      netrc.save
+      res = http.post('/oauth/token', data.to_json, {'Content-Type' => 'application/json'})
+
+      case res
+      when Net::HTTPSuccess
+        token = JSON.parse(res.body)['access_token']
+        netrc = Netrc.read
+        netrc[IDOBATA_URL.host] = email, token
+        netrc.save
+      when Net::HTTPUnauthorized
+        puts
+        abort 'Authentication is fail. You may wrong Email or Password.'
+      else
+        abort 'Initialize fail.'
```

Comment:
1. Is "Initialization failed" or "Failed to initialize" or more appropriate?
2. This will cause the user to get stuck, so we will need to present some useful information.